README.md

```diff
@@ -670,12 +670,12 @@ Target Block Rate: 2s (Same as C-Chain)
 
 ```
 Network Name: WAGMI
-RPC URL: https://api-wagmi.avax-test.network/rpc
+RPC URL: https://subnets.avax.network/wagmi/wagmi-chain-testnet/rpc
 Chain ID: 11111
 Symbol: WGM
 ```
 
-![metamask](./imgs/metamask.png)
```

Comment:
The block explorer in this image should be: https://subnets.avax.network/wagmi/wagmi-chain-testnet/explorer